axlearn/common/checkpointer_orbax_emergency_replicator.py

```diff
@@ -64,6 +58,7 @@ def _wait_for_file_to_disappear(f, timeout=300):
         if not f.exists():
             return True
         time.sleep(1)
+    logging.error("File %s did not dissappear in time.", f)
     return False
```

Comment:
throw on error here?
**General comment:** In all places where we know restore will not work properly it's better to throw, and let the job restart. Then either the problem will self-correct, or the job will enter crashback loop. Both are preferable to silently restarting training from step 1.